pushDocs.ps1

```diff
@@ -0,0 +1,37 @@
+param([string]$buildFolder, [string]$docSource, [string]$repoUrl, [string]$email, [string]$username, [string]$personalAccessToken)
```

Comment:
personalAccessToken is not used

Reply:
nice catch. it's now part of the urlRepo.